### PR TITLE
Reorganize abstract reviewing templates

### PR DESCRIPTION
**Pull request by pferreir**
Just moved around some templates, in order to make them a little bit easier to maintain.

### CODE REVIEW

File: indico/modules/events/abstracts/templates/_abstract/_common.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{% macro render_edited_hint(dt, user=None) %}
```
Comment (ThiefMaster) [inline indico/modules/events/abstracts/templates/_abstract/_common.html]: I wouldn't underscore the directory name, just the file name

File: indico/modules/events/abstracts/templates/_abstract/public.html [hunk 2]
```diff
@@ -1,21 +1,133 @@
...
-    {% else %}
-        <span class="abstract-comment-edited" title="{{ dt|format_human_date }}">
-            · {% trans %}edited{% endtrans %}
-        </span>
+{% macro _render_submission_info(abstract) %}
+    {% if abstract.submitted_contrib_type %}
+        <div class="info-line">
+            <label>
+                {% trans -%}
+                    Contribution type:
```
Comment (ThiefMaster) [inline indico/modules/events/abstracts/templates/_abstract/public.html]: fits nicely in one line with the trans stuff ;)

File: indico/modules/events/abstracts/templates/_abstract/review.html [hunk 3]
```diff
@@ -0,0 +1,85 @@
+{% from 'events/abstracts/_abstract/_common.html' import render_edited_hint %}
+
+{% macro render_abstract_reviews(abstract) %}
+    <div id="abstract-reviews" class="i-timeline">
+        {% set reviews = abstract.reviews|selectattr('track', 'equalto', none)|list %}
```
Comment (ThiefMaster) [inline indico/modules/events/abstracts/templates/_abstract/review.html]: `selectattr('track', 'none')` should be enough (there's a `none` test in jinja, so no need to use `equalto` for it)

File: indico/modules/events/abstracts/templates/_abstract/review.html [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+{% endmacro %}
+
+{% macro _render_review(review) %}
+    <div class="i-timeline-item">
+        <div class="i-timeline-item-label action abstract-comment-pic"></div>
+        <div class="i-timeline-item-box header-indicator">
+            <div class="i-box-header flex">
+                <div class="stretch">
+                    {% trans name=review.user.name, date=review.created_dt|format_human_date -%}
+                        <strong>{{ name }}</strong> reviewed on {{ date }}
```
Comment (ThiefMaster) [inline indico/modules/events/abstracts/templates/_abstract/review.html]: "foo reviewed on yesterday", "foo reviewed on today" :/

Comment (pferreir) [inline indico/modules/events/abstracts/templates/_abstract/review.html]: In this case I guess that removing the preposition would sound better. "Reviewed 21/10/2016" doesn't look that bad IMHO.

File: indico/modules/events/abstracts/templates/_abstract/_common.html (hunk 1 above)
Comment (pferreir) [inline indico/modules/events/abstracts/templates/_abstract/_common.html]: But there are non-underscored files inside. It's true that maybe underscoring the dir is a bit too much.

Comment (pferreir) [inline indico/modules/events/abstracts/templates/_abstract/_common.html]: /cc @OmeGak

**Comment by pferreir** [comment]
Merged in `42af67e`

File: indico/modules/events/abstracts/templates/abstract.html [hunk 5]
```diff
@@ -0,0 +1,18 @@
+{% extends 'events/abstracts/management/_base.html' %}
+
+{% from 'events/abstracts/_abstract/public.html' import render_abstract_public, render_abstract_comments, render_edited_hint %}
```
Comment (OmeGak) [inline indico/modules/events/abstracts/templates/abstract.html]: We are not using `render_edited_hint` in this template.

File: indico/modules/events/abstracts/templates/abstract.html [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+{% from 'events/abstracts/_abstract/judge.html' import render_decision_box %}
+{% from 'events/abstracts/_abstract/review.html' import render_abstract_reviews %}
+
+{% block content %}
+    {{ render_abstract_public(abstract, management=management) }}
+    <div id="abstract-private" class="i-timeline">
+        <div class="i-timetable-connect-down"></div>
+        <div class="i-timeline-separator"></div>
+        {{ render_abstract_comments(abstract) }}
```
Comment (OmeGak) [inline indico/modules/events/abstracts/templates/abstract.html]: We are importing `render_abstract_comments` from `public.html` but it's clearly something strictly non-public. I would move it somewhere else. Maybe `review.html`?

**Pull request closed without merging**